app/components/ExtraSettings.vue

```diff
@@ -2,27 +2,34 @@
 <div>
   <div class="section">
     <div class="section-content">
-      <p>{{ $t('If you are experiencing weird behavior, you can try deleting your cache directory.  This will result in you losing your scene configuration and settings, but can fix some stability issues.')}}</p>
+      <p>{{ $t('If you are experiencing weird behavior, you can try deleting your cache directory. This will result in you losing your scene configuration and settings, but can fix some stability issues.')}}</p>
```

Comment:
Kids these days and their single spaces after periods.  🤔

Reply:
double spaces (depending on style guide) haven't been proper english since at least 2003 and as early as the 80s